openidm-maintenance/src/main/java/org/forgerock/openidm/maintenance/impl/UpdateLogServiceImpl.java

```diff
@@ -138,4 +138,8 @@ public void updateUpdate(UpdateLogEntry entry) throws ResourceException {
                 new UpdateContext(ContextUtil.createInternalContext()),
                 Requests.newUpdateRequest(updateStore, entry.getId(), entry.toJson()));
     }
+    
+    protected void bindConnectionFactory(IDMConnectionFactory connectionFactory) {
```

Comment:
@wahyupratomoicsg got error if dont add this?

Reply:
If I run "Build with dependencies" from netbeans, yeah they find this method